src/main/java/seedu/edulog/logic/commands/UnmarkNameCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.edulog.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.edulog.commons.util.ToStringBuilder;
+import seedu.edulog.logic.Messages;
+import seedu.edulog.logic.commands.exceptions.CommandException;
+import seedu.edulog.model.Model;
+import seedu.edulog.model.student.Name;
+import seedu.edulog.model.student.Student;
+
+/**
+ * Unmarks a student identified using it's displayed index from the edulog book.
+ */
+public class UnmarkNameCommand extends UnmarkCommand {
+
+    private final Name targetName;
+
+    public UnmarkNameCommand(Name targetName) {
+        this.targetName = targetName;
+    }
+
+    @Override
```

Comment:
I like how you used used the @override tag, so that the compiler can check whether we are actually overriding a method